Added info file for version 3.5.3

rafaelbeirigo opened the pull request:
Added 3.5.3

rafaelbeirigo committed:
Changed everything

politza commented:
What is this ?

<div>-------- Ursprüngliche Nachricht --------</div><div>Von: Rafael Lemes Beirigo <notifications@github.com> </div><div>Datum:03.02.2018  2:46  (GMT+01:00) </div><div>An: politza/python-info <python-info@noreply.github.com> </div><div>Cc: Subscribed <subscribed@noreply.github.com> </div><div>Betreff: [politza/python-info] Changed everything (#1) </div><div>
</div>You can view, comment on, or merge this pull request online at:

  https://github.com/politza/python-info/pull/1

Commit Summary

Changed everything
File Changes

A python-3.5.3.info.gz (0)
Patch Links:

https://github.com/politza/python-info/pull/1.patch
https://github.com/politza/python-info/pull/1.diff
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.

rafaelbeirigo commented:
Greetings!

I forked your repository and added the info file for version 3.5.3 (the one I'm using at the moment). As your repository appears on the wiki, I thought you would want to include it, so I sent you the pull request. I hope you don't mind!

Best regards,
Rafael

rafaelbeirigo committed:
Add info file for several versions

Versions:

3.3.3
3.3.4
3.3.5
3.3.6
3.3.7
3.4.0
3.4.1
3.4.2
3.4.3
3.4.4
3.4.5
3.4.6
3.4.7
3.5.0
3.5.1
3.5.2
3.5.4
3.6.0
3.6.1
3.6.2
3.6.3
3.6.4

rafaelbeirigo committed:
Add installation instructions
